cni/pkg/ambient/net_linux.go

```diff
@@ -193,6 +193,9 @@ func (s *Server) delPodEbpfOnNode(ip string) error {
 		ifIndex = veth.Attrs().Index
 	}
 
+	if force {
+		return s.ebpfServer.RemovePod([]netip.Addr{ipAddr}, uint32(ifIndex))
+	}
```

Comment:
Mostly a nit - why the use of a separate public method for this versus adding an extra flag to RedirectArgs?

Reply:
Hmm, it's  a bit tricky here. Since channel messaging will be terminated with context.Done(channel message mechanism could guarantee the sequence at least), so we need a "force"-like method for cleanup.

Reply:
Ah, ok - that's fine, I wasn't sure.

Long term that makes me wonder if the request-based "server" model might not actually be that helpful versus simpler public methods, but that's out of scope here.